include/vrv/doc.h

```diff
@@ -254,10 +260,10 @@ class Doc : public Object {
 
     /**
      * Casts off the entire document, with options for obeying breaks.
-     * @param useSb - true to use the sb from the document.
+     * @param useSb - 1 to use the sb from the document, 2 to sometimes us the sb from the document.
      * @param usePg - true to use the pb from the document.
      */
-    void CastOffDocBase(bool useSb, bool usePb);
+    void CastOffDocBase(int useSb, bool usePb);
```

Comment:
I would prefer to have an extra parameter `bool smart = false` rather than changing useSb to int because this is a bit cryptic.

Reply:
Agreed (and changed in b7cf23747f8189402666411faf9b2e8f40b88609).

Though it seems a little strange to have three boolean variables for a function which (as implemented) only has three different execution paths (rather than `2^3`, that is, 8). I guess this is just one of the development consequences of modifying the function prototype before modifying any of the code.